cmd/ddev/cmd/dev_sequelpro.go

```diff
@@ -6,69 +6,107 @@ import (
 	"os"
 	"os/exec"
 	"path"
-	"strconv"
 
+	"github.com/drud/ddev/pkg/appports"
 	"github.com/drud/ddev/pkg/plugins/platform"
 	"github.com/drud/ddev/pkg/util"
 	"github.com/drud/drud-go/utils/dockerutil"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	"runtime"
 )
 
-// LocalDevSequelproCmd represents the sequelpro command
-var LocalDevSequelproCmd = &cobra.Command{
+// SequelproLoc is where we expect to find the sequel pro.app
+// It's global so it can be mocked in testing.
+var SequelproLoc = "/Applications/sequel pro.app"
+
+// localDevSequelproCmd represents the sequelpro command
+var localDevSequelproCmd = &cobra.Command{
 	Use:   "sequelpro",
 	Short: "Easily connect local site to sequelpro",
-	Long:  `A helper command for easily using sequelpro with a drud app that has been initialized locally.`,
+	Long:  `A helper command for easily using sequelpro (OSX database browser) with a ddev app that has been initialized locally.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		app, err := getActiveApp()
+		out, err := handleSequelProCommand(SequelproLoc, args)
 		if err != nil {
-			log.Fatalf("Could not find an active ddev configuration, have you run 'ddev config'?: %v", err)
+			log.Fatalf("Could not run sequelpro command: %s", err)
 		}
+		util.Success(out)
+	},
+}
 
-		nameContainer := fmt.Sprintf("%s-db", app.ContainerName())
+// handleSequelProCommand() is the "real" handler for the real command
+func handleSequelProCommand(appLocation string, args []string) (string, error) {
```

Comment:
I absolutely want Cobra to do reasonable things and handle all validation. Can't really understand how we're not there yet :) Every argument handling package since the beginning of Unix time has done this better. And yes, this was added explicitly and only for testing the extra args problem.